app/lib/features/space/pages/related_spaces_page.dart

```diff
@@ -41,8 +41,8 @@ class RelatedSpacesPage extends ConsumerWidget {
           ...spaces.when(
             data: (spaces) {
               final widthCount =
-                  (MediaQuery.of(context).size.width ~/ 600).toInt();
-              const int minCount = 2;
+                  (MediaQuery.of(context).size.width ~/ 300).toInt();
+              const int minCount = 3;
```

Comment:
if we are reusing the static numbers, shouldn't we put them into a central location? maybe even our custom theme?

Reply:
minCount will be different for each grid like spaces and members have different minimum count.